eth2near/ethashproof/cmd/relayer/main.go

```diff
@@ -61,6 +61,13 @@ func main() {
 		}
 	}
 
+    // Remove previous epoch
+	if epoch > 0 {
+		previousEpoch := epoch - 1
+		os.Remove(ethash.PathToDAG(previousEpoch, ethash.DefaultDir))
+		os.Remove(ethashproof.PathToCache(previousEpoch))
+	}
+
```

Comment:
```suggestion
        // Remove previous epoch
	if epoch > 1 {
		previousEpoch := epoch - 2
		os.Remove(ethash.PathToDAG(previousEpoch, ethash.DefaultDir))
		os.Remove(ethashproof.PathToCache(previousEpoch))
	}

```

I would not remove the previous, but previous to previous epoch instead! If there is a reorg around the epoch change it would be needed to recompute the previous epoch.

Do we need to check if the file exists before trying to remove it? (not sure how os.Remove works in go)